tests/test_to_process.py

```diff
@@ -0,0 +1,82 @@
+import os
+import platform
+import sys
+import time
+from functools import partial
+
+import pytest
+
+from anyio import CancelScope, create_task_group, fail_after, to_process, wait_all_tasks_blocked
+
+pytestmark = pytest.mark.anyio
+
+
+@pytest.fixture(autouse=True)
+def check_compatibility(anyio_backend_name):
+    if anyio_backend_name == 'asyncio':
+        if platform.system() == 'Windows' and sys.version_info < (3, 8):
+            pytest.skip('Python < 3.8 uses SelectorEventLoop by default and it does not support '
+                        'subprocesses')
```

Comment:
If Python < 3.8 is not supported on windows, will an informative error be raised if `to_process.run_sync` is called?

Reply:
Asyncio raises a `NotImplementedError`. I wonder if we need a better error message. Maybe we'll cross that bridge when we come to it? It may be that nobody even cares :)